app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,15 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  height: 70px;
+  border-top: 1px solid $govuk-border-colour;
+  margin-bottom: 16px;
 }
 
 .gem-c-pagination__link {
   @extend %govuk-link;
   display: block;
-  padding: govuk-spacing(3);
+  padding: 15px 0 20px 0;
```

Comment:
Can we use the govuk-frontend spacing scale here? Also why is the padding now irregular?

Reply:
have applied the spacing scale, the irregular padding is as the design